README.md

```diff
@@ -437,6 +438,17 @@ repository.records.first.sets
 
 Return an `Array` of `String` [set specs](#fieldhandsetspec) indicating set memberships of this record.
 
+#### `Fieldhand::Record#dump`
```

Comment:
I think it's OK because we don't need symmetry in these two cases:

The contract with `metadata` is "give me the metadata for this record" and Fieldhand makes the decision to return it to you as a string (rather than another object or a tree of XML nodes).

For the record itself, you want to say "give me this entire record as an XML string" so `to_xml` seems appropriate.